test/specs/eazework.spec.ts

```diff
@@ -0,0 +1,21 @@
+import eazeworkPage from "../pageobjects/eazework.page.js";
+import credentails from "../data/crdentails.json" assert {type:"json"};
+
+describe('Feature: Leave Request Save in Draft', () => {
+    beforeEach(async () => {
+        await browser.url("https://www.eazework.com/Login/");
+        await browser.maximizeWindow();
+    });
+
+    it('should log in successfully', async () => {
+       
+         await eazeworkPage.logintoEazework("credex","Abhishek.gautam","#abhishek@123");
```

Comment:
put all the creds in the .env file and make a params.ts file where you will define variables and that will be used in the spec files.
e.g.

//params.ts


module.exports = {
  url: process.env.URL,
  userName: process.env.USERNAME,
  }
// and in the spec file 
1. import the params file
2. Use object destructuring to get the url or the creds, like
const {url} = params.url